DS-Session-2-Assignment/src/question2/Queue.java

```diff
@@ -0,0 +1,25 @@
+package question2;
+
+/**
+ * This interface represents the operations which are to be performed on a queue
+ * @author Jugal
+ * Dated 07/31/2019
+ */
+public interface Queue {
+	
+	//Method for inserting data in the queue
+	public void enqueue(int data) throws Exception;
```

Comment:
do not throw Exception, just print the message to the user and continue your program.